package.json

```diff
@@ -29,8 +29,9 @@
     "install": "node-pre-gyp install --fallback-to-build"
   },
   "dependencies": {
-    "nan": "2.6.2",
-    "node-pre-gyp": "0.6.36"
+    "bindings": "^1.3.0",
```

Comment:
Please pin these dependencies.

Reply:
No. Just remove the caret. "^1.3.0" becomes "1.3.0" and so on.

Reply:
Ok just done. Can you check?